backend/controllers/faq.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const cors = require('cors')
+const faqRouter = express.Router()
+
+var mongodb = require('mongodb');
+var Faq = require('../models/FAQ');
+
+faqRouter.use(cors())
+faqRouter.use(bodyParser.json());
+faqRouter.use(bodyParser.urlencoded({
+  extended: true
+}));
+
+faqRouter.get('/list', function(req, res) {
+	Faq.find(function(err, faqs) {
+		if (err) {
+			res.json(err);
+		}
+		faqs = faqs.map(q => q.toJSONFor());
+		res.json(faqs);
+	});
+})
+
+faqRouter.delete('/delete', function(req, res) {
+	var ids = req.body.faqIDs.map(function(id) {
+		return new mongodb.ObjectID(id); 	
+	});
+
+	Faq.deleteMany({_id: {$in: ids}}, function(err) {
+		if (err) {
+			res.send(err);
+		}
+	});
+	res.send('Deleted questions!');
```

Comment:
It might be pedantic but I think a better response would be a JSON object.
E.g.

`{"message": "success"}`

Or something to that degree.

Reply:
kk will do